<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInput</class>
 <widget class="QDialog" name="DialogInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seleziona modalità</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Con quale modalità si preferisce procedere?</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="tastieraRadioButton">
     <property name="text">
      <string>Input da tastiera (senza usare server)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="serverRadioButton">
     <property name="text">
      <string>Input da tastiera usando il il server per il calcolo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="databaseRadioButton">
     <property name="text">
      <string>Input acquisito da Database PostGres</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okPushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closePushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
